src/lib/components/list/QItem.svelte

```diff
@@ -76,6 +77,7 @@
     {...Q.classes}
     tabindex={isClickable ? tabindex || 0 : undefined}
     aria-disabled={isActionable && disabled ? true : undefined}
+    data-quaff
```

Comment:
Why a second time?

Reply:
What do you mean? Like "why in QItem if we use it on QList too"?

Reply:
Yeah

Reply:
Because users should be able to attach tooltips to any QItem imho, not only the full list